examples/pump-app/telink/src/AppTask.cpp

```diff
@@ -34,7 +34,7 @@ AppTask AppTask::sAppTask;
 CHIP_ERROR AppTask::Init(void)
 {
     SetExampleButtonCallbacks(StartActionEventHandler);
-    InitCommonParts();
+    TEMPORARY_RETURN_IGNORED InitCommonParts();
```

Comment:
The `AppTask::Init` function returns CHIP_ERROR, so why not to forward error from `InitCommonParts()` up the call stack?